docs/reference/commandline/run.md

```diff
@@ -529,54 +588,15 @@ Use Docker's `--restart` to specify a container's *restart policy*. A restart
 policy controls whether the Docker daemon restarts a container after exit.
 Docker supports the following restart policies:
 
-<table>
```

Comment:
No, that was a different (more complex) table. It was the one that had to do with volume options.